app/src/main/java/com/macbitsgoa/events/maps/MapsActivity.java

```diff
@@ -0,0 +1,161 @@
+package com.macbitsgoa.events.maps;
+
+
+import android.os.Bundle;
+import android.util.Log;
+
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.database.ValueEventListener;
+import com.macbitsgoa.events.R;
+
+import com.google.android.gms.maps.CameraUpdateFactory;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.OnMapReadyCallback;
+import com.google.android.gms.maps.SupportMapFragment;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.MarkerOptions;
+
+import java.util.ArrayList;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.FragmentActivity;
+
+/**
+ * Google Maps Activity showing Event Markers.
+ *
+ * @author Rajath Reghunath [rrgodhorus@gmail.com]
+ */
+
+@SuppressWarnings("DuplicateStringLiteralInspection")
+public class MapsActivity extends FragmentActivity implements OnMapReadyCallback {
+
+    private static final int MAP_INIT_CAP = 15;  //initial capacity for number of venues
+    private static final String MAP_CENTRAL_LOC = "/locations/central";
+    private static final String MAP_ZOOM_SETTINGS = "/locations/zoom";
+    private static final String MAP_MARKERS_LOC = "/locations/locations";
+    private static final float MAP_DEFAULT_ZOOM = 16.0f;
+    private static final Venue MAP_DEFAULT_CENTRAL = new Venue("15.392199","73.878609","BITS GOA");
+    private static final String FIREBASE_ERROR_TAG = "MapsActivity";
+    private static final String FIREBASE_ERROR_STRING = "Firebase data retrieval failed";
+    private static final String FIREBASE_STRING_LATITUDE = "latitude";
+    private static final String FIREBASE_STRING_LONGITUDE = "longitude";
+    private static final String FIREBASE_STRING_VENUE_NAME = "venueName";
+
+    private float zoomLevel = MAP_DEFAULT_ZOOM;
+    @SuppressWarnings("TypeMayBeWeakened")
+    private final ArrayList<Venue> venues  = new ArrayList<>(MAP_INIT_CAP);
+    private GoogleMap map;
+
+    @Override
+    protected void onCreate(final Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_maps);
+
+        // Obtain the SupportMapFragment and get notified when the map is ready to be used.
+        final SupportMapFragment mapFragment = (SupportMapFragment) getSupportFragmentManager()
+                .findFragmentById(R.id.map);
+        if (mapFragment != null) {
+            mapFragment.getMapAsync(this);
+        }
+    }
+
+
+    /**
+     * Manipulates the map once available.
+     * This callback is triggered when the map is ready to be used.
+     * This is where we can add markers or lines, add listeners or move the camera. In this case,
+     * we just add a marker near Sydney, Australia.
```

Comment:
delete this line

Reply:
The whole comment or just the sydney part?

Reply:
sydney part